docs/content/docs/core-concept/transform.md

```diff
@@ -28,5 +28,245 @@ under the License.
 **Transform** module helps users delete and expand data columns based on the data columns in the table.      
 What's more, it also helps users filter some unnecessary data during the synchronization process.
 
+# Parameters
+To describe a transform, the follows are required:
+
+| Parameter    | Meaning                                            | Optional/Required |
+|--------------|----------------------------------------------------|-------------------|
+| source-table | Source table id, supports regular expressions      | required          |
+| projection   | Projection rule, supports syntax similar to the select clause in SQL        | optional          |
+| filter       | Filter rule, supports syntax similar to the where clause in SQL        | optional          |
+| primary-keys | Sink table primary keys, it is separated by English commas        | optional          |
+| partition-keys | Sink table partition keys, it is separated by English commas        | optional          |
+| table-options | Sink table options, when automatically creating a table, it is added to the configuration of the table creation statement  | optional          |
+| description  | Transform rule description(a default value provided) | optional          |
```

Comment:
```suggestion
| description  | Transform rule description (a default description will be used if not specified) | optional          |
```

Does it mean if user didn't specify a transform description, CDC will generate a default one to indicate that transform rule?